agent/internal/request.go

```diff
@@ -179,7 +179,11 @@ func (ctx *HTTPRequestRecord) UserSecurityResponse() http.Handler {
 	if !exists {
 		return nil
 	}
-	ctx.lastUserSecurityResponseHandler = actor.NewUserActionHTTPHandler(action, userID)
+	var err error
+	ctx.lastUserSecurityResponseHandler, err = actor.NewUserActionHTTPHandler(action, userID)
+	if err != nil {
+		agent.logger.Error(sqerrors.Wrap(err, "user security response"))
```

Comment:
Include the actual User in the log output.

Reply:
If there's a way to identify security responses (uuid?) maybe include that too.

Reply:
done